### PR TITLE
fix(dashboard): correct redeployment button

### PR DESCRIPTION
**Pull request by szilarddoro**
This is a follow-up PR for https://github.com/nhost/nhost/pull/1503. Also, fixes an issue where the "Live" badge was shown for every deployment that was the first on the page on the "Deployments" page.

Preview: https://dashboard-staging-git-fix-retrigger-deployment-status-nhost.vercel.app/

### CODE REVIEW

File: dashboard/src/components/ui/StatusCircle.tsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import clsx from 'clsx';
+import { twMerge } from 'tailwind-merge';
 
 export type DeploymentStatus =
   | 'DEPLOYING'
   | 'DEPLOYED'
   | 'FAILED'
+  | 'PENDING'
```
Comment (guicurcio) [inline dashboard/src/components/ui/StatusCircle.tsx:7]: These three states match the `deploymentStatus` column in the `deployments` table:

<table><tbody><tr><td>DEPLOYING</td></tr><tr><td>DEPLOYED</td></tr><tr><td>FAILED</td></tr></tbody></table>

do the new ones 'PENDING' & 'SCHEDULED' are also new states in this column? @elephant3 I haven't seen them in staging, do we mutate a given deployment to go from, say, PENDING -> FAILED? I think we could also look up into creating these states as an enum-like similar to what we have with the `app_states`

Comment (elephant3) [reply dashboard/src/components/ui/StatusCircle.tsx:7]: Yes, they are - https://github.com/nhost/watchtower/blob/ce804295c13a4d0067e20a941123ae2de77fb1ce/pkg/hasura/deployments_notificator.go#L18-L23

`PENDING` -> `FAILED` is possible, as well as `SCHEDULED` -> `FAILED` if they get stuck

Comment (guicurcio) [reply dashboard/src/components/ui/StatusCircle.tsx:7]: Ah, got it. Thanks for clarifying. I'll open up an issue in console-next in the event we want to have something like the \`**app\_states\`** table to get a grasp for all possible states of a deployment :+1:

**Review by guicurcio — APPROVED** [review]

**Review by elitan — APPROVED** [review]

File: dashboard/src/components/ui/StatusCircle.tsx (hunk 1 above)
Comment (7koma95) [reply dashboard/src/components/ui/StatusCircle.tsx:7]: > Ah, got it. Thanks for clarifying. I'll open up an issue in console-next in the event we want to have something like the \`**app\_states\`** table to get a grasp for all possible states of a deployment :+1:

**Review by 7koma95 — APPROVED** [review]

**Pull request merged**